src/Microsoft.VisualStudio.Services.Agent/ConfigurationStore.cs

```diff
@@ -46,6 +46,12 @@ public sealed class AgentSettings
 
         [DataMember(EmitDefaultValue = false)]
         public int DeploymentGroupId { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public string ProjectId { get; set; }
```

Comment:
why do we need store projectid? we already store projectName. also projectId is unreadable most of time.